src/schema/v2/partner.ts

```diff
@@ -339,7 +339,7 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
             missing_priority_metadata: args.missingPriorityMetadata,
             artist_id: args.artistID || undefined,
             page,
-            published: args.published ?? true,
+            published: args.includeUnpublished ? false : true,
```

Comment:
Ah the upstream API takes a boolean `published` param that supports true/false? Tricksky!